app/src/components/HomePage/HomePage.tsx

```diff
@@ -126,26 +142,22 @@ export default function HomePage({
             population={population}
           />
 
-          <Box
-            className="flex"
-            justifySelf={"center"}
-            style={{ maxWidth: "90vw" }}
-          >
+          <Box className="flex mx-auto mt-[80px] max-w-full w-[1090px]">
             <VStack align="start">
               <InventoryPreferencesCard t={t} isPublic={isPublic} />
-              <Text
-                color="content.primary"
-                fontWeight="bold"
-                lineHeight="24px"
-                fontSize="headline.sm"
-                fontFamily="heading"
-                fontStyle="normal"
-                my="48px"
-              >
-                {t("ghg-emissions-inventory-in-year", {
-                  year: inventory?.year,
-                })}
-              </Text>
+              {/*<Text*/}
+              {/*  color="content.primary"*/}
+              {/*  fontWeight="bold"*/}
+              {/*  lineHeight="24px"*/}
+              {/*  fontSize="headline.sm"*/}
+              {/*  fontFamily="heading"*/}
+              {/*  fontStyle="normal"*/}
+              {/*  my="48px"*/}
+              {/*>*/}
+              {/*  {t("ghg-emissions-inventory-in-year", {*/}
+              {/*    year: inventory?.year,*/}
+              {/*  })}*/}
+              {/*</Text>*/}
```

Comment:
Can you remove the commented code once you're done or use one block comment for the whole block at least?